pkg/koordlet/koordlet.go

```diff
@@ -105,7 +105,13 @@ func NewDaemon(config *config.Configuration) (Daemon, error) {
 		}
 		klog.Infof("can not detect cgroup driver from 'kubepods' cgroup name")
 
-		port := int(statesInformer.GetNode().Status.DaemonEndpoints.KubeletEndpoint.Port)
+		node := statesInformer.GetNode()
```

Comment:
thanks for your fix the issue, but why can't get Node, have you found the root cause?

Reply:
Root cause seems to be #407